sriov/common/utils.py

```diff
@@ -219,15 +219,12 @@ def vfs_created(ssh_obj, pf_interface, num_vfs, timeout = 10):
     Returns:
         True: all VFs are created
         False: not all VFs are created before timeout exceeded
-
-    Raises:
-        Exception on command failure
     """
     cmd = "ls -d /sys/class/net/" + pf_interface + "v* | wc -w"
     for i in range(timeout):
         code, out, err = ssh_obj.execute(cmd)
         if code != 0:
-            raise Exception(err)
+            continue
         if int(out[0].strip()) == num_vfs:
             return True
         time.sleep(1)
```

Comment:
sleep should happen before continue; otherwise there is no actual wait

Reply:
Good catch, this is updated on https://github.com/redhat-partner-solutions/intel-sriov-test/pull/32/commits/70e794ce807521477673f5cf9f60a4f1ee818e8b